src/linpred.jl

```diff
@@ -362,7 +362,7 @@ fitted(m::LinPredModel) = m.rr.mu
 predict(mm::LinPredModel) = fitted(mm)
 residuals(obj::LinPredModel) = residuals(obj.rr)
 
-function formula(obj::LinPredModel)
+function StatsModels.formula(obj::LinPredModel)
```

Comment:
While we are at it. When is it called. When I do:
```
julia> formula(lm(x, y))
ERROR: type LinearModel has no field fr

julia> formula(glm(x, y, Normal()))
ERROR: type GeneralizedLinearModel has no field fr
```
other methods are called.

Do we have tests for different cases when formula is not present?

Reply:
Hmmm, will investigate. I thought we caught this when Milan removed TableRegressionModel.

Reply:
On current master:

```julia
julia> formula(lm(ones(10, 1),  randn(10)))
ERROR: ArgumentError: model was fitted without a formula
Stacktrace:
 [1] formula(obj::LinearModel{GLM.LmResp{Vector{Float64}}, GLM.DensePredChol{Float64, LinearAlgebra.CholeskyPivoted{Float64, Matrix{Float64}, Vector{Int64}}}})
   @ GLM ~/Code/GLM.jl/src/linpred.jl:366
 [2] top-level scope
   @ REPL[13]:1

julia> formula(glm(ones(10, 1),  randn(10), Normal()))
ERROR: ArgumentError: model was fitted without a formula
Stacktrace:
 [1] formula(obj::GeneralizedLinearModel{GLM.GlmResp{Vector{Float64}, Normal{Float64}, IdentityLink}, GLM.DensePredChol{Float64, LinearAlgebra.CholeskyPivoted{Float64, Matrix{Float64}, Vector{Int64}}}})
   @ GLM ~/Code/GLM.jl/src/linpred.jl:366
 [2] top-level scope
   @ REPL[14]:1
```

(will have to keep this in mind for the backport to 1.x where we still have TableRegressionModel)